{{ .ProjectSnake }}/MODULE.bazel

```diff
@@ -13,6 +13,7 @@ bazel_dep(name = "aspect_rules_ts", version = "3.0.0-rc1")
 {{- end }}
 {{- if .Computed.python }}
 bazel_dep(name = "rules_python", version = "0.32.2")
+bazel_dep(name = "rules_python_gazelle_plugin", version = "0.32.2")
```

Comment:
Is it worth templating the version into a variable to ensure they are always in sync?

Reply:
I don't think so because renovate is managing this file nicely, and it will see both modules updated

Reply:
Neat that it picks it up 👍